spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+require "codeclimate-test-reporter"
+CodeClimate::TestReporter.start
+
 require 'simplecov'
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
```

Comment:
I think that we want to run simplecov only if the CodeClimate environment variable is *not* set, so this should be something like:

```ruby
if ENV.key?('CODE_CLIMATE_WHATEVER')
  require "codeclimate-test-reporter"
  CodeClimate::TestReporter.start
else
  require 'simplecov'
  require 'simplecov-rcov'
  SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
end
```